src/ut/fakesnmp.cpp

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file fakesnmp.cpp Fake SNMP infrastructure for UT.
+ *
+ * project clearwater - ims in the cloud
```

Comment:
This copyright statement is all in lowercase - can you copy/paste it from a good one (e.g. https://github.com/Metaswitch/sprout/blob/dev/sprout/mmtelasplugin.cpp) instead?

(Some of our existing copyright statements are all in lowercase - we think a vim command went badly wrong once - but we should make sure all the new files are good.)